src/app/beer_garden/config.py

```diff
@@ -1261,6 +1261,11 @@ def _parse_args(args: Sequence[str]) -> Tuple[YapconfSpec, dict]:
                 },
             },
         },
+        "allow_update_systems": {
+            "type": "bool",
+            "default": False,
+            "description": "Allow updating the commands of a a non-dev system",
```

Comment:
You've got a double "a" here:  `a a non-dev` system.  I might just reword this as:

> Allow commands of non-dev systems to be updated